lambda/README.md

```diff
@@ -0,0 +1,31 @@
+# Mattermost load-test Lambda function
+
+This is a quick demo using a Lambda function to load-test some of the edge-cases that our primary tool is unable to do. For example:
+- Websocket reconnect.
+- Large number of users joining a team in a short time.
+- Large number of users joining a channel in a short time.
+- Possible others.
+
+## Required Libraries and Dependencies
+
+- `puppeteer-core`: Provides the API to control Chromium.
+- The `@sparticuz/chromium` library is provided as a layer.
+- Node.js 14 or higher recommended
```

Comment:
I'd probably include an `.nvmrc` file to align with our main repos (currently at `20.11`). To note, anything below v20 is pas end-of-life I believe.